service/api.go

```diff
@@ -560,9 +562,27 @@ func (as *APIServer) validateProjectConfig(w http.ResponseWriter, r *http.Reques
 		return
 	}
 
-	errs := validator.CheckYamlStrict(input.ProjectYaml)
+	errs := validator.ValidationErrors{}
+	if input.ProjectID != "" {
```

Comment:
Should we add a warning like "validation will proceed without checking project settings" if projectID != ""?

Reply:
Should be for == "" correct?